README.md

```diff
@@ -66,11 +66,11 @@ Sending a donation using our links below is **extremely helpful** in offsetting
 ### Alternative support
 If you'd rather not [donate](https://pi-hole.net/donate/) (_which is okay!_), there are other ways you can help support us:
 - [Patreon](https://patreon.com/pihole) _Become a patron for rewards_
-- [Digital Ocean](http://www.digitalocean.com/?refcode=344d234950e1) _affiliate link_
+- [Digital Ocean](https://www.digitalocean.com/?refcode=344d234950e1) _affiliate link_
 - [Stickermule](https://www.stickermule.com/unlock?ref_id=9127301701&utm_medium=link&utm_source=invite) _earn a $10 credit after your first purchase_
-- [Amazon](http://www.amazon.com/exec/obidos/redirect-home/pihole09-20) _affiliate link_
+- [Amazon](https://www.amazon.com/exec/obidos/redirect-home/pihole09-20) _affiliate link_
 - [DNS Made Easy](https://cp.dnsmadeeasy.com/u/133706) _affiliate link_
-- [Vultr](http://www.vultr.com/?ref=7190426) _affiliate link_
+- [Vultr](https://www.vultr.com/?ref=7190426) _affiliate link_
```

Comment:
Can you remove this line completely? No one has ever signed up for Vultr with our afflink. We don't even use them anymore.

Reply:
Should we remove them from the docs repo also?

Reply:
Yes please.

Reply:
Maybe this is a good chance to streamline the links here and in the docs repo?

https://docs.pi-hole.net/#pi-hole-is-free-but-powered-by-your-support

Reply:
Sounds good.